services/ui-src/src/components/index.ts

```diff
@@ -19,7 +19,9 @@ export { Banner } from "./banners/Banner";
 export { PreviewBanner } from "./banners/PreviewBanner";
 // cards
 export { Card } from "./cards/Card";
-export { EntityCard } from "./cards/EntityCard";
+export { EntityCard } from "./cards/EntityCard/EntityCard";
+export { EntityCardTopText } from "./cards/EntityCard/EntityCardTopText";
+export { EntityCardBottomText } from "./cards/EntityCard/EntityCardBottomText";
```

Comment:
Suggestion: Maybe `Section` instead of `Text`? Idk you know I'm bad at naming things.